CharlieBackend.Api/CharlieBackend.Api.csproj

```diff
@@ -5,6 +5,11 @@
     <UserSecretsId>77c89e00-81d7-4d64-9c07-8a105a6066bb</UserSecretsId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
do not ignore this warning. It will help to document all public facing properties and API's

Reply:
Ok.